packages/core-modules/contracts/modules/CoreElectionModule.sol

```diff
@@ -152,24 +157,30 @@ contract CoreElectionModule is IElectionModule, ElectionStorage, OwnableMixin {
 
         uint votePower = ERC20(store.electionTokenAddress).balanceOf(msg.sender);
 
-        for (uint i = 0; i < candidates.length; i++) {
-            address candidate = candidates[i];
+        address[] memory candidates = new address[](sortedCandidates.length);
 
-            if (electionData.nomineeIndexes[candidate] == 0) {
-                revert InvalidCandidate(candidate);
+        for (uint i = 0; i < sortedCandidates.length; i++) {
+            address candidate = sortedCandidates[i];
+            uint position = positions[i];
+
+            // Validate positions repetition
+            if (candidates[position] != address(0)) {
+                revert InvalidCandidatesPositions();
```

Comment:
This error could be `DuplicateCandidatePriority`.

Reply:
Ready:  https://github.com/Synthetixio/synthetix-v3/pull/572/files#diff-69650d386e562b4dac992709531657fe63fedd176cde00f236b7b6271333c70eR160-R161